core/jvm/src/main/scala/zio/sql/typetag.scala

```diff
@@ -9,32 +9,32 @@ trait TypeTagModule {
 
   type TypeTagExtension[+A]
 
-  sealed trait TypeTag[+A] {
+  sealed trait TypeTag[A] {
```

Comment:
the covariance makes things much safer RE casting (i.e. you can't do it incorrectly using `cast`) so I think this should stay.